README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01: Self-Introduction
+My name is **Xuefei Gao** and you can just call me **Ian**, which is easier to pronounce and remember. I am a raising second year and I plan to major in Math and Econ. I am also a fellow of SISRM and I am going to work with Professor Susan Stokes this summer.
+
+![Photo](Self_Portrait.png)
+
+Here are some of my interests:
+1. Music
+  * Rock
+  * Classical
+2. Basketball
+3. Sneakers
+
+Here is my homepage of Github: https://github.com/IanGGG
```

Comment:
It doesn't really matter, since the link worked anyway, but if you wanted to hyperlink a word you could use the code [Github] (https://github.com/IanGGG).